frontend/src/features/judging/components/JudgeSlot.tsx

```
@@ -0,0 +1,125 @@
+        animate={entranceVariant.animate}
+        transition={entranceVariant.transition}
+        draggable={false}
+      />
+
+      {/* スコアパネル */}
+      <div
+        data-testid="judge-desk-score"
+        data-lit={isLit ? 'true' : 'false'}
+        className={`judge-desk-panel glass-panel ${neonClass.border}`}
+        aria-label={buildScoreAriaLabel(judge, scoreLabel)}
+      >
```

Biome (2.4.4)

[error] The ARIA attribute 'aria-label' is not supported by this element.

(lint/a11y/useAriaPropsSupportedByRole)